pyproject.toml

```
@@ -45,6 +45,16 @@
     name = "Contributor-facing changes"
     showcontent = true
 
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  # Essentials
+  "setuptools >= 45",
+
+  # Plugins
+  "setuptools_scm[toml] >= 6.2",
+  "setuptools_scm_git_archive>=1.1",
```

For some reason, the surrounding whitespaces are missing here:
```suggestion
  "setuptools_scm_git_archive >= 1.1",
```